PoSSapi/src/Domain/Entities/Payment.cs

```diff
@@ -14,5 +14,5 @@ public class Payment
     public decimal Tip { get; set; }
     public PaymentOption PaymentOptions { get; set; }
     public PaymentStatus Status { get; set; }
-    public DateTime CompletionTime { get; set; }
+    public DateTime TimeWhenCompleted { get; set; }
```

Comment:
The `Payment` table in the DB still has the column with name `CompletionTime`, so a new migration should be made to change the table column's name